index.bs

```diff
@@ -201,9 +201,9 @@ The following steps are taken:
 <emu-alg>
 1. Let _loader_ be *this* value.
 1. If Type(_loader_) is not Object, throw a *TypeError* exception.
-1. If _loader_ does not have a [[Registry]] internal slot throw a *TypeError* exception.
 1. Return the result of transforming Resolve(_loader_, _name_, _referrer_) with a fulfillment handler that, when called with argument _key_, runs the following steps:
-  1. Return RequestReady(_loader_, _key_).
+  1. Let _entry_ be EnsureRegistered(_loader_, _key_).
+  1. Return _entry_.load().
```

Comment:
This is another place where we're calling `.load()` instead of an abstract operation.